network-api/networkapi/templates/partials/primary_nav.html

```diff
@@ -34,7 +34,7 @@
                 <div class="d-flex align-items-center flex-wrap">
 
                   {% block burger_and_logo %}
-                    <button class="burger {% if page.zen_nav == False %} hidden-lg-up ml-md-0 {% endif %}" aria-label="{% trans 'Open menu' %}">
+                    <button class="burger {% if page.zen_nav == False or page.zen_nav == null %} hidden-lg-up ml-md-0 {% endif %}" aria-label="{% trans 'Open menu' %}">
```

Comment:
Can you check for "not True" in Python instead of the double condition?

Reply:
sure can